plugins/catalog-backend/src/ingestion/processors/codeowners/scm.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const scmCodeOwnersPaths: Record<string, string[]> = {
+  bitbucket: ['.bitbucket/CODEOWNERS'],
+  gitlab: ['.gitlab/CODEOWNERS', 'docs/CODEOWNERS'],
+  github: ['.github/CODEOWNERS', 'docs/CODEOWNERS'],
```

Comment:
Yep, your right. I'll add back `root`. I as hoping to avoid having to make multiple calls to the API though. I have not tested this, but guessing each one even if it's a 404 impacts the rate limits. So this it's not ideal to have to call multiple times. I haven't really experimented with the reader.search function so not sure it's feasible to use this for that.

Reply:
Yeah that's a viable concern. Hmmm I should try to boot up my etag-respecting experiment again ... which would make these requests not count against quota

Reply:
Maybe a follow up could be to have this an optional config preference for something like `integration.[provider].[host].codeowner.paths`?

Reply:
Hm! Yeah I guess that could actually make sense even though it's such a narrow thing.